libcaf_core/caf/detail/json.hpp

```diff
@@ -467,11 +467,9 @@ using file_parser_state = parser_state<std::istreambuf_iterator<char>>;
 // Parses the input string and makes a deep copy of all strings.
 value* parse(string_parser_state& ps, monotonic_buffer_resource* storage);
 
-// Parses the input string and makes a deep copy of all strings.
-value* parse(file_parser_state& ps, monotonic_buffer_resource* storage);
-
 // Parses the input file and makes a deep copy of all strings.
-value* parse(string_parser_state& ps, monotonic_buffer_resource* storage);
+CAF_CORE_EXPORT value* parse(file_parser_state& ps,
+                             monotonic_buffer_resource* storage);
```

Comment:
Had to return this function back as exported since it's used in unit tests. When I remove the export the tests won't link when `caf-core` is used as a shared library. This happened locally and in the pipeline from the previous commit.